Upgrade application from LB2 to LB3

jisoolee opened the pull request:
Update for loopback version 3

jisoolee committed:
Upgrade application from LB2 to LB3

BREAKING CHANGE: upgrade the application from LB2 to LB3 using guide
provided by https://loopback.io/doc/en/lb3/Migrating-to-3.0.html.
Upgrade dependencies accordingly.

Co-authored-by: Biniam Admikew <biniam@ca.ibm.com>

sam-github commented:
`fix(dep): ...` suggests a patch release to me, but this bumps the package major. Just a note, I'm not involved in maintaining this stuff.

bajtos reviewed (approved):
I am not familiar with this codebase. I don't see any obvious problems and the tests pass, so LGTM.

Kudos for bumping-up the major version :+1:

rmg commented on package.json:
```diff
@@ -12,24 +12,26 @@
   },
   "devDependencies": {
-    "eslint": "^1.x",
+    "eslint": "^5.16.0",
```
Is it necessary to update `eslint`?

b-admike replied:
For posterity, as discussed, this is because of security vulnerabilities reported by `npm audit`.

Commit:
chore: upgrade dependencies

upgrade dependencies and fix lint issues introduced
as part of the new version of eslint-config-strongloop
